src/pages/api/og-image.jsx

```diff
@@ -0,0 +1,139 @@
+import { ImageResponse } from 'next/server';
+
+export const runtime = 'edge';
```

Comment:
Let's try to avoid using old pages directory for this. Here is a example of how can you make it work with app dir.

https://github.com/leerob/leerob.io/blob/main/app/og/route.tsx

Also since og images are very sensitive to url changes, let's don't make that path(current /api/og-image) global if it's at this moment used only in docs. So I'd suggest to narrow it down.